src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeCache.targets

```diff
@@ -23,11 +23,132 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <Target Name="ComposeCache"
-          DependsOnTargets="PrepareForComposeCache;
-                            RestoreForComposeCache;
-                            ComputeAndCopyFilesToCacheDirectory"/>
+          DependsOnTargets="_InitializeBasicProps;
+                            PrepareForComposeCache;
+                            PrepOptimizer;
+                            CacheWorkerMain;
+                            CacheFinalizer;"/>
+
+
+  <!--
+    ============================================================
+                                        CacheWorkerMain
+
+   Spins of the compose mappers
+    ============================================================
+    -->
+
+  <Target Name="CacheWorkerMain">
+    
+    <CreateItem Include="$(AdditionalProjects)">
+      <Output ItemName="_AllProjects" TaskParameter="Include" />
+    </CreateItem>
+    
+    <ItemGroup>
+      <_AllProjects Include ="$(MSBuildProjectFullPath)"/>
+    </ItemGroup>
+    <MSBuild Projects="%(_AllProjects.Identity)"
+                 Targets="CacheWorkerMapper"
+                 Properties="ComposeWorkingDir=$(ComposeWorkingDir);
+                             PublishDir=$(PublishDir);
+                             TargetFramework=$(_TFM);
+                             _JitPath=$(_JitPath);
+                             _Crossgen=$(_Crossgen);
+                             SkipUnchangedFiles=$(SkipUnchangedFiles);
+                             PreserveCacheLayout=$(PreserveCacheLayout)"
+                  ContinueOnError="WarnAndContinue"  />
+   </Target>
+  <!--
+    ============================================================
+                                        CacheWorkerMapper
+
+   Spins of the compose mappers
+    ============================================================
+    -->
+
+  <Target Name="CacheWorkerMapper">
+   
+    <ItemGroup>
+      <PackageToRestore Include ="@(PackageReference)" Condition="'%(PackageReference.IsImplicitlyDefined)' != 'true'"/>
```

Comment:
Will remove this and replace it with DisableImplicitFrameworkReferences=true in the invocation of CacheWorkerMapper